paper-drawer-panel.html

```diff
@@ -239,12 +239,12 @@
     <iron-selector
         attr-for-selected="id"
         class$="[[_computeIronSelectorClass(narrow, transition, dragging, rightDrawer)]]"
-        on-iron-activate="onSelect"
+        activate-event=""
```

Comment:
`iron-selector` has a default value for `activateEvent` https://github.com/PolymerElements/iron-selector/blob/master/iron-selectable.html#L57.  We would want to set to empty string to remove it.